### PR TITLE
Acceptance test to logout user from the account page

### PR DESCRIPTION
**Pull request by jasson99**
## Description
Add acceptance test to logout user from the account page

## Related Issue

-Related to issue #2566 

## Motivation and Context
The feature for logging out from the account page was implemented in phoenix but no tests were written for that feature. This PR adds acceptance test scenarios for that feature.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingAcceptShares</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/6832/

<p>





</p></details>

File: tests/acceptance/pageObjects/accountPage.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+     *
+     * @returns {object}
+     */
+    getAccountInformation: async function () {
+      const accountInformation = []
+      let accountInfoElementIds = []
+      await this.waitForElementVisible('@accountInformationElements')
+      await this.api.elements('@accountInformationElements', function (result) {
+        accountInfoElementIds = result.value
+      })
```
Comment (kiranparajuli589) [inline tests/acceptance/pageObjects/accountPage.js]: await this.waitForElementVisible('@accountInformationElements')
                .api.elements('@accountInformationElements', function (result) {
                           accountInfoElementIds = result.value
                 })

Comment (jasson99) [reply tests/acceptance/pageObjects/accountPage.js]: done

File: tests/acceptance/features/webUIAccount/logout.feature [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Feature: Logout users
+  As a user
+  I want to be able to logout of my account
+  So that I can protect my work and identity and be assured of privacy
+
+  Scenario:logging out
+    Given user "user1" has been created with default attributes
+    And user "user1" has logged in using the webUI
+    When the user browses to the account page
+    And the user clicks the logout button
```
Comment (individual-it) [inline tests/acceptance/features/webUIAccount/logout.feature]: don't use `click` e.g. on a mobile device the user cannot click

```suggestion
    And the user logs out
```

Comment (jasson99) [reply tests/acceptance/features/webUIAccount/logout.feature]: Thankyou.

**Pull request merged**